core/src/test/scala/org/apache/spark/util/SparkUncaughtExceptionHandlerSuite.scala

```diff
@@ -74,6 +82,46 @@ object ThrowableTypes extends Enumeration {
   val SparkFatalOutOfMemoryError = ThrowableTypesVal("SparkFatalException(OutOfMemoryError)",
     new SparkFatalException(new OutOfMemoryError))
 
+  // SPARK-50034: If there is a fatal error in the cause chain,
+  // we should also identify that fatal error and exit with the
+  // correct exit code.
+  val NestedOOMError = ThrowableTypesVal(
+    "NestedFatalError",
+    new RuntimeException("Nonfatal Level 1",
+      new RuntimeException("Nonfatal Level 2",
+        new RuntimeException("Nonfatal Level 3",
+          new OutOfMemoryError())))
+  )
+
+  val NestedSparkFatalException = ThrowableTypesVal(
+    "NestedMixedFatalErrors",
```

Comment:
What is mixed fatal error?

Reply:
Changed to `NestedSparkFatalException`. I changed the variable name but forgot to change the name inside `ThrowableTypesVal`. Fixed.